package.json

```diff
@@ -54,12 +52,10 @@
     "react-router": "^5.2.0",
     "react-table": "^7.6.3",
     "sass": "^1.27.0",
-    "sass-loader": "^10.0.4",
```

Comment:
Is the sass-loader not necessary anymore?

Reply:
No since we changed the import for bootstrap to be via a cdn

Reply:
But we still need the `sass` pacakge?